diff --git a/node_modules/@walletconnect/modal/dist/_types/src/client.d.ts b/node_modules/@walletconnect/modal/dist/_types/src/client.d.ts
index fc7c092..61491fe 100644
--- a/node_modules/@walletconnect/modal/dist/_types/src/client.d.ts
+++ b/node_modules/@walletconnect/modal/dist/_types/src/client.d.ts
@@ -1,3 +1,4 @@
+//@ts-nocheck
 import type { ConfigCtrlState, ThemeCtrlState } from '@walletconnect/modal-core';
 /**
  * Types
